supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -13,14 +13,21 @@ When the format _day Month year_ causes a presentation or clarity issue, use _Mo
 [[conversational-style]]
 == Conversational style
 
-Follow _The IBM Style Guide_ guidance of _less conversational_ style in most cases. 
+Follow _The IBM Style Guide_ guidance of _less conversational_ style in most cases.
 
 As needed, adjust the conversational to _fairly conversational_ for an audience of new users or _least conversational_ for API documentation and other very experienced audiences.
 
 .Example: Less conversational style
 
 Red Hat Enterprise Linux 8 delivers a stable, secure, and consistent foundation across hybrid cloud deployments with the tools needed to deliver workloads faster with less effort.
 
+[[contractions]]
+== Contractions
+
+Avoid contractions in product documentation to leave no ambiguity and to make it easier for translation and international audiences.
+
+If you are writing quickstart or other content that uses a more informal xref:#conversational-style[conversational style] (_fairly conversational_ or _more conversational_), you may use contractions. In this case, follow the guidance in _The IBM Style Guide_ on using contractions.
```

Comment:
This is also a bit nitpicky but 'may' is not recommended by IBM, in this case 'can' would work better.

Reply:
@ndeevy You know, I usually never use "may" but did in this case because of the ISG saying to use it to indicate permission. As opposed to "can" meaning that you have the ability.

> can  verb
> Use to indicate ability. Use instead of "could". To indicate possibility, use "might". To indicate permission, use "may".
> In most cases, use "can" or "might", not "may".

Let me know if you still think we ought to use "can" instead, and if so I'll update.

Reply:
Ooh, I stand corrected. It is permission so makes sense here. nitpick:reversed! :-]